defs.h

```diff
@@ -29,12 +29,9 @@
 #ifndef IDA_GARBAGE
 #define IDA_GARBAGE
 
-#include <stdint.h>
-
-// note to self: only works for x86, originally used this way by the devs
 inline void memset32(void *s, unsigned int c, size_t n)
 {
-	uint32_t *p = (uint32_t*)s;
+	unsigned int *p = (unsigned int *)s;

```

Comment:
Is this also correct on 64-bit platforms?

Reply:
Well assuming an integer was defined as 8-bytes than much of the game would break. It's a long way before it's cross compatible between 32/64. A 64-bit platform should treat the project as 32-bit, at least for the time being. Did this change cause problems on your setup?

Reply:
> Well assuming an integer was defined as 8-bytes than much of the game would break. It's a long way before it's cross compatible between 32/64. A 64-bit platform should treat the project as 32-bit, at least for the time being.

Sounds reasonable.

> Did this change cause problems on your setup?

No, it works just fine. In the Makefile GCC is configured to compile a 32-bit executable.

So, lets keep it like this for now. Then we can do a swoop through the entire project at a later point, if we feel to compile a true 64-bit version.